src/Cli/dotnet/commands/dotnet-sln/remove/Program.cs

```diff
@@ -38,15 +38,14 @@ public override int Execute()
 
             try
             {
-                var relativeProjectPaths = _projects.Select(p =>
-                {
-                    var fullPath = Path.GetFullPath(p);
-                    return Path.GetRelativePath(
+                var relativeProjectPaths = _projects
+                    .Select(p => Path.GetFullPath(p))
```

Comment:
What's the point of this part? Seems like it does the exact same thing but slower.

Reply:
Seemed cleaner and easier to read in my mind, but can revert it :)